site/src/pages/contact.page.tsx

```diff
@@ -29,12 +30,22 @@ const Partners: NextPage = () => {
             textAlign="center"
             maxWidth={750}
             variant="bpSubtitle"
-            sx={{ fontSize: 20 }}
+            sx={{ fontSize: 20, marginBottom: 2 }}
           >
             If you’re working on an existing or upcoming block editor, or would
             otherwise like to contribute to the new standard, fill in some
             details below and we’ll be in touch.
           </Typography>
+          <Typography
+            textAlign="center"
+            maxWidth={750}
+            variant="bpSubtitle"
+            sx={{ fontSize: 20, marginBottom: 2 }}
+          >
+            You can also{" "}
+            <Link href="mailto:help@blockprotocol.org">email us</Link> or{" "}
+            <Link href="https://hash.ai/discord">chat to us on Discord</Link>.
```

Comment:
There is a separate Discord link for the BP site. We already use it in the footer IIRC

Reply:
```suggestion
            <Link href="https://discord.gg/PefPteFe5j">chat to us on Discord</Link>.
```